### PR TITLE
hack: prod cron job

### PR DESCRIPTION
**Pull request by acud**
do not merge

### CODE REVIEW

File: pkg/check/pushsync/check_chunks.go [hunk 1]
```diff
@@ -1,102 +1,103 @@
...
-			skipPeers := []swarm.Address{closestAddress, uploaderAddr}
-			// chunk should be replicated at least once either during forwarding or after storing
-			for range overlays {
-				name, address, err := chunk.ClosestNodeFromMap(overlays, skipPeers...)
-				skipPeers = append(skipPeers, address)
-				if err != nil {
-					continue
-				}
-				synced, err = clients[name].HasChunk(ctx, ref)
+		//if len(clients) == 0 {
```
Comment (mrekucci) [inline pkg/check/pushsync/check_chunks.go:68]: This leftover should be deleted.

**Review by mrekucci — APPROVED** [review]
Except for the linting error LGTM.

**Comment by acud** [comment]
run with:
```
make binary && ./dist/beekeeper check --cluster-name=mainnet --checks=pushsync-chunks --metrics-enabled=false --create-cluster=false                                                                                              
```
once push gateway is deployed remove the `metrics-enabled` flag to enable metrics

File: config/mainnet.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    resources-limit-memory: 2Gi
+    resources-request-cpu: 750m
+    resources-request-memory: 1Gi
+    restart-policy: "Always"
+    update-strategy: "RollingUpdate"
+
+# bee-configs defines Bee configuration that can be assigned to node-groups
+# bee-configs may inherit it's configuration from already defined bee-config and override specific fields from it
+bee-configs:
+  mainnet:
```
Comment (acud) [inline config/mainnet.yaml:67]:
```
bee-configs:
  testnet:
    _inherit: "default"
    bootnode: "/dnsaddr/testnet.ethswarm.org"
  mainnet:
    _inherit: "default"
    bootnodes: "/dnsaddr/mainnet.ethswarm.org"
```

**Pull request closed without merging**